lisa/wlgen/rta.py

```diff
@@ -329,23 +329,39 @@ def _calibrate(cls, target, res_dir):
         logger = cls.get_logger()
 
         # Create calibration task
-        max_rtprio = int(target.execute('ulimit -Hr').split('\r')[0])
-        logger.debug('Max RT prio: %d', max_rtprio)
+        if target.is_rooted:
+            max_rtprio = int(target.execute('ulimit -Hr').split('\r')[0])
+            logger.debug('Max RT prio: %d', max_rtprio)
 
-        if max_rtprio > 10:
-            max_rtprio = 10
+            if max_rtprio > 10:
+                max_rtprio = 10
+
+            sched = {
+                'policy': 'FIFO',
+                'prio': max_rtprio,
+            }
+        else:
+            logger.warning('Will use CFS scheduler class instead of RT since the target is not rooted')
```

Comment:
Nit: unlikely to diverge but hey :man_shrugging: 
```suggestion
            logger.warning('Will use default scheduler class instead of RT since the target is not rooted')
```